src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig

```diff
@@ -24,7 +24,7 @@
             <td>
                 {{ log.message|format_log_message(log.context) }}
                 {% if log.context ?? false %}
-                    <span class="block text-muted prewrap m-t-5">{{ log.context|json_encode(constant('JSON_PRETTY_PRINT') b-or constant('JSON_UNESCAPED_UNICODE') b-or constant('JSON_UNESCAPED_SLASHES')) }}</span>
+                    <pre class="text-muted prewrap m-t-5">{{ log.context|json_encode(constant('JSON_PRETTY_PRINT') b-or constant('JSON_UNESCAPED_UNICODE') b-or constant('JSON_UNESCAPED_SLASHES')) }}</pre>
```

Comment:
do you still need `prewrap` here ? If no, this could be removed (and removing the class from the cSS as your PR is adding it)

Reply:
Yes, I think it's needed because when outputting long lines (e.g. long class namespaces) the `<pre>` doesn't break lines and makes the debug page hugely wide.